apps/src/templates/sectionProgressV2/coming-soon-banner.module.scss

```
@@ -26,6 +28,7 @@
   }
 
   & .bannerText {
+    flex: 1 1 80%;
```

```suggestion
    flex-grow: 1;
```
Do we need to set flex-shrink or flex-basis or can we just use flex-grow?

---

Thanks! Tested this change locally 👍